### ISSUE TITLE
Expected number of all hosts (0) to be remote +1 (0)

### ISSUE THREAD

**Issue by kaloyan-raev** [minio/minio#10783]
An unexpected error/warning is printed to the console right after starting the MinIO gateway.

## Expected Behavior

No error/warning should be printed.

## Current Behavior

Here is the complete error/warning with the stack trace.

```
Error: WARNING: Expected number of all hosts (0) to be remote +1 (0) (*errors.errorString)
       4: /home/kaloyan/git/minio/cmd/peer-rest-client.go:866:cmd.newPeerRestClients()
       3: /home/kaloyan/git/minio/cmd/notification.go:1292:cmd.NewNotificationSys()
       2: /home/kaloyan/git/minio/cmd/server-main.go:160:cmd.newAllSubsystems()
       1: /home/kaloyan/git/minio/cmd/gateway-main.go:304:cmd.StartGateway()
```

It does not seem to affect any functionality, but it confuses users that something might be wrong.

## Steps to Reproduce (for bugs)

1. Start the MinIO S3 Gateway
2. The error is printed to the console.

## Context

This happens also to our Tardigrade S3 Gateway after we updated it to the latest master of MinIO.

## Regression

Yes, this is a regression.

## Your Environment

* Version used (`minio --version`): minio version DEVELOPMENT.GOGET
* Server setup and configuration:
* Operating System and version (`uname -a`): Linux t480 5.8.13-200.fc32.x86_64 #1 SMP Thu Oct 1 21:49:42 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

**Comment by vadmeste** [comment]
Thanks for reporting this, we will quickly fix it.

**Comment by vadmeste** [comment]
This will be fixed here, https://github.com/minio/minio/pull/10777/files

**Comment by harshavardhana** [comment]
Fixed in #10777
